src/Cargo.lock

```diff
@@ -87,7 +87,7 @@ dependencies = [
 
 [[package]]
 name = "atty"
-version = "0.2.11"
+version = "0.2.10"
```

Comment:
weird, why did this happen?

Reply:
Uh..I have no idea. I should revert that.

Reply:
But you did not... the downgrade landed.

Reply:
Submitted fix as https://github.com/rust-lang/rust/pull/52779